docs/.vuepress/redirects

```diff
@@ -40,6 +40,7 @@
 /guides/guides/addressing/ /how-to/address-ipfs-on-web
 /guides/guides/install/ /install
 /how-to/host-single-page-site /how-to/websites-on-ipfs/single-page-website
+how-to/browser-tools-frameworks /how-to/ipfs-on-the-web/

```

Comment:
Does this need a leading `/` like the others?

Reply:
Probably not, even though it appends the trailing slash if not there